.github/workflows/build.yml

```diff
@@ -50,7 +50,7 @@ jobs:
     name: Java ${{ matrix.java }}
     env:
       OS: ${{ matrix.os }}
-      JAVA: ${{ matrix.java-version }}
+      JAVA: ${{ matrix.java }}
```

Comment:
It's not clear to me what the previous version of this was doing ..... shouldn't `matrix.java-version` have always been null? :|

Reply:
I think so. Not sure the environment variable is even used actually...